### DISCUSSION TITLE
RasperryMatic zeigt seit heute ein verfügbares Update von RedMatic 7.2.1 auf 7.2.0 an?!?

### DISCUSSION THREAD

**Discussion by bvol** [rdmtc/RedMatic#544] [Q&A]
Vielleicht bin ich zu doof, aber seit heute zeigt mein Raspberrymatic 3.63.9.20220430 ein verfügbares Update von Redmatic auf 7.2.0 an obwohl ich 7.2.1 schon länger am laufen habe. Geht es noch anderen so und ist eventuell ein Grund dafür bekannt?

![image](https://user-images.githubusercontent.com/45370257/167465944-964c74f2-a75f-4bc2-b447-563116c6a404.png)

**Comment by blackflash73** [comment]
Ja, geht uns genauso! ;)
-> Issues https://github.com/rdmtc/RedMatic/issues/543

**Reply by bvol** [reply]
seit heute erkennt raspberrymatic wieder die 7.2.1 als latest, warum auch immer

**Comment by hobbyquaker** [comment]
Die Update-Prüfung verglich Versionen nur auf Ungleichheit; in RedMatic 9 wird korrekt verglichen, Vorabversionen werden nicht angeboten. Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
